src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -12,18 +12,26 @@
     [broadfcui.utils :as utils]
     ))
 
-(defn- render-wizard-breadcrumbs [{:keys [library-schema page-num]}]
+(defn- render-wizard-breadcrumbs [{:keys [library-schema page-num pages-seen]}]
   (let [pages (:wizard library-schema)]
-    [:div {:style {:flex "0 0 250px" :backgroundColor "white" :border style/standard-line}}
-     [:ul {}
+    [:div {:style {:flex "0 0 250px" :backgroundColor "white"
+                   :border style/standard-line :padding "1rem"}}
+     [:div {:style {:paddingBottom "0.5rem" :fontWeight (when (< page-num (count pages)) "bold")}} "Catalog "]
+     [:ul {:style {:WebkitMarginBefore 0 :WebkitMarginAfter 0}}
       (map-indexed
        (fn [index {:keys [title]}]
          (let [this (= index page-num)]
-           [:li {:style {:margin "0.5em 0.5em 0.5em 0"
+           [:li {:style {:paddingBottom "0.5rem"
                          :fontWeight (when this "bold")
-                         :color (when-not this (:text-lighter style/colors))}}
-            title]))
-       (conj pages {:title "Summary"}))]]))
+                         :color (when-not (or this (contains? pages-seen index)) (:text-lighter style/colors))}}
+            title])) pages)]
+      [:div {:style {:paddingBottom "0.5rem"
+                     :color (when-not (contains? pages-seen (count pages)) (:text-lighter style/colors))
+                     :fontWeight (when (= page-num (count pages)) "bold")}}
+       "Discoverability"]
+      [:div {:style {:color (when-not (contains? pages-seen (+ 1 (count pages))) (:text-lighter style/colors))
+                     :fontWeight (when (> page-num (count pages)) "bold")}}
+       "Summary"]]))
```

Comment:
we wanted to do that also, until we realized that you can't really go straight to the 3rd page because it will be different things depending on what you selected for the 2nd page... so we just punted on this.